src/Instrumentation/Psr15/src/Psr15Instrumentation.php

```diff
@@ -24,10 +23,18 @@
  */
 class Psr15Instrumentation
 {
+    private static $rootSpan;
```

Comment:
This can be moved out from `Psr15Instrumentation` then we will not depend on it but on something generic